persistence-cassandra/core/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraKeyspaceConfig.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.cassandra
+
+import akka.event.LoggingAdapter
+import com.typesafe.config.Config
+
+private[lagom] object CassandraKeyspaceConfig {
+
+  def validateKeyspace(namespace: String)(implicit config: Config, log: LoggingAdapter): Unit =
+    validateKeyspace(namespace, s"$namespace.defaults")
+
+  def validateKeyspace(namespace: String, defaultNamespace: String)(implicit config: Config, log: LoggingAdapter): Unit = {
+    val path = s"$namespace.keyspace"
+    if (!config.hasPath(path)) {
+      log.error("Configuration for [{}] must be set in application.conf", path)
+    } else if (log.isWarningEnabled) {
+      val keyspace = config.getString(path)
+      val defaultKeyspace = config.getString(s"$defaultNamespace.keyspace")
+      if (keyspace == defaultKeyspace) {
```

Comment:
There's 3 possible situations:
1. defaults == non_defaults && defaults == reference_values_in_akka
2. defaults == non_defaults && defaults != reference_values_in_akka
3. defaults != non_defaults

(1) is the situation we observed in #578 that leads to all services using the same keyspace
(2) is considered invalid by this PR but could be usable (like you mentioned earlier) 
(3) is valid.

I think the root problem of avoiding many services use the same keyspaces and this PR will only cover half of the problem: users may still force all keyspaces to a single, reused keyspace.

Maybe the scope of this PR should detecting that values are different from akka.defaults. Lagom defaults (generated from project name) are enough for preventing keyspace collision. 

Is that correct?